db/schema.rb

```diff
@@ -11,17 +11,26 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150804180349) do
+ActiveRecord::Schema.define(version: 20150806184710) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "transactions", force: :cascade do |t|
+    t.date     "date",       null: false
+    t.integer  "amount",     null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer  "user_id"
+    t.string   "category"
+  end
+
   create_table "users", force: :cascade do |t|
     t.string   "name",            null: false
     t.string   "address"
     t.string   "phone"
     t.string   "email",           null: false
-    t.string   "password_digest"
+    t.string   "password_digest", null: false
```

Comment:
this needs to remain the way it was, i removed the null false because it will mess things up